### ISSUE TITLE
Repeated "Nice work" 

### ISSUE THREAD

**Issue by evelinec** [OpenLiberty/openliberty.io#1012]
Reported by @yeekangc: 
"Nice work" is repeated under "Nice work! Where to next?", which is unnecessary?

![image](https://user-images.githubusercontent.com/33134427/53199145-4fddca00-35ec-11e9-821c-1b77908953c3.png)

**Comment by steven1046** [comment]
This was the intended design with the left and right side of the end of the guide matching the 2 parts of the header, @cajaygle do you have any thoughts?

**Comment by cajaygle** [comment]
<img width="1062" alt="screen shot 2019-02-21 at 3 35 09 pm" src="https://user-images.githubusercontent.com/27516391/53199928-4e150600-35ee-11e9-904d-ac20f1ff3d6a.png">

And on the right, "Where to next?" is repeated.  😱   I think repeating "Nice work!" is fine and fits with the text.  People don't necessarily read section headers.

In the Docker guide, look at the section title and the first line of text.  https://qa-guides-multipane.mybluemix.net/guides/docker.html#building-the-image  Should we remove "To build your image" from the first line of text because it repeats the section title?  No.

**Comment by yeekangc** [comment]
I still don't see why we need "Nice work!" in the text.  The section header is big and clear enough.  It makes the content cleaner and easier to manage too.

**Comment by yeekangc** [comment]
So, this "Nice work!" in the text is appended by the build somehow?  And, we mapped "== Great work! You're done!" in the source to "Nice work! Where to next?"  If so, I think we need to discuss these further.

**Comment by steven1046** [comment]
We prepend the "Nice work!" before the first paragraph in the end of guide, it's an option to add it to every adoc and the remove the javascript that adds it.

**Comment by yeekangc** [comment]
What was the reason for doing that, @steven1046?  Can you please elaborate?

I think the section title should match what's shown after the build too.  So, "== Great work! You're done!" should be "== Nice work! Where to next?" if that's what we want to have.

Fwiw, none of these is critical to switch over to multi-pane completely.

FYI, @gkwan-ibm.

**Comment by steven1046** [comment]
It was part of the design feedback from the user testing, where the user didn't know there was an end of guide section. See the conversation regarding the heading title here: https://ibm-cloud.slack.com/archives/C5Y532PB3/p1549048012000200

**Comment by yeekangc** [comment]
No issue with the change in the title itself.  Just that I want the source to match the new title.  Leaving it with the old title and having the build to convert will most likely need to confusion down the road.  Need to fix in the source (asciidocs). @gkwan-ibm, please follow up.

I can be persuaded regarding the repeat of "Nice work!" but I wouldn't want it to be prepended by the build.  Makes it cleaner and easier to understand and maintain.

**Comment by gkwan-ibm** [comment]
reopen for further discussion

**Comment by mbroz2** [comment]
Since this work is not necessary for the release of the redesigned static guides (and it's starting to sound like any changes are not user-facing as far as site visitors go), I'm tagging this as 'needs discussion' and moving it to the icebox.

**Comment by yeekangc** [comment]
Yes, @mbroz2.  This isn't directly user facing but it affects guide authors.  It needs further discussion and putting it in the Icebox for now is fine.  We will pick it up as soon as ready.  Thank you.

**Comment by steven1046** [comment]
Can we just remove the `Nice work!` from the text in the end of guide section since it appears in the yellow section header?

**Comment by natalie-bernhard** [comment]
Conclusions from site meeting:

- [ ] Site code - Remove JS logic that attaches the `Nice Job` in the body. Just use the content from the adoc file
- [ ] Title - Use what is in the adoc file which is generally “Great work! You’re done!” for all guides
- [ ] License sentence - Should not be using internal link blue coloring
  - Idea: Suggestion is to move the license sentence lower
  - Idea: Another section just says “License”, like “What did you think of your guide?”
  - Idea: Generalize the sentence to speak to all guides
  - Make the license a link

**Comment by gkwan-ibm** [comment]
need to coordinate when publish the change because the guides may need to be updated too

**Comment by navaneethsnair1** [comment]
Hi @gkwan-ibm , I think the first and second points by @natalie-bernhard could be implemented easily. We can discuss further on what to be done on the 3rd point

**Comment by gkwan-ibm** [comment]
My suggestion:

> ...
> **Need help?**
> ...
> **Like Open Liberty? Star our repo on GitHub.**
> ...
> **License**
> The Testing microservices with the Arquillian managed container guide by Open Liberty is licensed under [CC BY-ND 4.0](https://github.com/OpenLiberty/guide-arquillian-managed/blob/prod/LICENSE)

where
- "Testing microservices with the Arquillian managed container" is the guide title
- The link points to the LICENSE file in the guide GH repo
  - also follows Natalie's suggestion "Should not be using internal link blue coloring "

**Comment by navaneethsnair1** [comment]
Currently:
<img width="1534" height="1136" alt="Image" src="https://github.com/user-attachments/assets/196f0740-2abf-4394-9d3d-1b1f9eeb91a7" />

Suggested improvement by @IsmathBadsha : (cc: @gkwan-ibm )
<img width="1536" height="1382" alt="Image" src="https://github.com/user-attachments/assets/d22e047c-64ed-4f42-b90a-33a20958f2bc" />

@mbroz2 's suggestion:
<img width="3404" height="1468" alt="Image" src="https://github.com/user-attachments/assets/58c555be-81e0-4cc0-9486-3d099bc3bf2a" />
